Sources/Context.swift

```diff
@@ -3,8 +3,12 @@ public class Context {
   var dictionaries: [[String: Any?]]
 
   public let environment: Environment
+  
+  public var errorReporter: ErrorReporter {
+    return environment.errorReporter
+  }
 
-  init(dictionary: [String: Any]? = nil, environment: Environment? = nil) {
+  init(dictionary: [String: Any?]? = nil, environment: Environment? = nil) {
```

Comment:
I've seen this change in multiple pull requests. Wanna split this out into a separate PR for discussion?

Reply:
it's in #166 , I just based all other branches on its branch so that tests don't randomly fail, that's why you see them in each pr =)